database/certificate.go

```diff
@@ -388,6 +388,38 @@ func (db *DB) GetAllCertsInStore(store string) (out []certificate.Certificate, e
 	}
 }
 
+func (db *DB) GetAllCertIDsIssuedBy(sha256 string)(out []int64, err error) {
```

Comment:
Two more comments:
1. add a comment string that describes what the method does
2. does this return only end-entities or intermediates as well? I think the naming of the function should make that as unambiguous as possible. Maybe `GetEndEntityCertsForIssuerBySha256Fingerprint`. It's a mouthful, but there would be no confusion as to what it does.